src/models/chan.js

```diff
@@ -185,7 +185,8 @@ Chan.prototype.writeUserLog = function(client, msg) {
 	}
 
 	// Is this particular message or channel loggable
-	if (!msg.isLoggable() || !this.isLoggable()) {
+	// exception for notices so they get logged even when sent to the lobby
+	if (!msg.isLoggable() || (!this.isLoggable() && msg.type !== "notice")) {
```

Comment:
I think if we change this to `if (!msg.isLoggable() && !this.isLoggable()) {`

It should work just as fine.